### DISCUSSION TITLE
Nova Notification Unread colours should follow notification type.

### DISCUSSION THREAD

**Discussion by AdrianwithaW** [laravel/nova-issues#4628] [Ideas & Feature Requests]
- Laravel Version: 9.21.6
- Nova Version: 4.12.12
- PHP Version: 8.1.6
- Database Driver & Version: MySQL 5.7.12
- Operating System and Version: Ubuntu 18.04.3
- Browser type and version: Any
- Reproduction Repository: https://github.com/###/###

### Description:

In calling a Nova Notification using the following:
```
NovaNotification::make()
  ->message("{Redacted}")
  ->action('View', "/resources/{redacted}/{redacted}")
  ->icon('info-circle')
  ->type('success')
```

the notification works exactly as expected but appears red as an error rather than green as a successful message. The database notification is being stored correctly, but any browser displays the notification as an error… there is an additional notification from earlier in time for the same resource (with the same action URL) in the nova notifications table that has been sent to the same User (i.e. Notifiable model) from calling the same Notification method earlier but as an error which has now been fixed - is this a limitation of the notification feature that a notification needs to have some sort of uniqueness?

**Comment by RealityGamin** [comment]
Support..

Can we also make a button for "Delete all notifications"?
